### PR TITLE
fix swarm db export import

### PR DESCRIPTION
**Pull request by nonsense**
This PR is addressing https://github.com/ethersphere/go-ethereum/issues/231

TODO
- [x] add automated test for export/import
- [ ] maybe refactor `testCluster` interface.

### CODE REVIEW

File: cmd/swarm/export_test.go [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+	var info swarm.Info
+	if err := cluster.Nodes[0].Client.Call(&info, "bzz_info"); err != nil {
+		t.Fatal(err)
+	}
+
+	cluster.Stop()
+	defer cluster.Cleanup()
+
+	// generate an export.tar
+	exportCmd := runSwarm(t, "db", "export", info.Path+"/chunks", info.Path+"/export.tar", info.BzzKey[2:])
```
Comment (janos) [inline cmd/swarm/export_test.go]: I am suggesting `strings.TrimPrefix(info.BzzKey, "0x")` instead `info.BzzKey[2:]` for more clear intension and correctness. The same goes for `info2.BzzKey` and `info2.BzzAccount` in this function.

File: cmd/swarm/export_test.go [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+	res, err := http.Get(cluster2.Nodes[0].URL + "/bzz:/" + hash)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if res.StatusCode != 200 {
+		t.Fatalf("expected HTTP status %d, got %s", 200, res.Status)
+	}
+
+	//TODO: compare res with generated random file
```
Comment (janos) [inline cmd/swarm/export_test.go]: Something like this:

```
	h := md5.New()
	upLen, err := io.Copy(h, f)
	if err != nil {
		t.Fatal(err)
	}
	upHash := h.Sum(nil)
	h.Reset()
	downLen, err := io.Copy(h, res.Body)
	if err != nil {
		t.Fatal(err)
	}
	downHash := h.Sum(nil)

	if !bytes.Equal(upHash, downHash) || upLen != downLen {
		t.Fatalf("downloaded imported file md5=%x (length %v) is not the same as the generated one mp5=%x (length %v)", downHash, downLen, upHash, upLen)
	}
```

**Review by janos — COMMENTED** [review]
LGTM, only the file validation in TestCLISwarmExportImport should be added.

File: cmd/swarm/export_test.go (hunk 1 above)
Comment (nonsense) [reply cmd/swarm/export_test.go]: OK, makes sense.

File: cmd/swarm/export_test.go (hunk 2 above)
Comment (nonsense) [reply cmd/swarm/export_test.go]: Thank you :)

**Review by janos — APPROVED** [review]
👍

**Pull request merged**